data/load-seed-data.js

```diff
@@ -11,7 +11,7 @@ async function run() {
   try {
     await client.connect();
 
-    const users = await Promise.all(
+    const chords = await Promise.all(
```

Comment:
This part actually should just be left alone as `users` -> we'll work on user Auth next week so this block of code and table will make more sense then.